swagger-spec/collection_submission_actions/detail.yaml

```
@@ -0,0 +1,73 @@
+# /collection_submission_actions/{collection_submission_id}/
+get:
+  summary: Retrieve a Collection Submission Action
+  description: >-
+
+    #### Permissions
+
+    This information is only available to an admin on the Collection in question.
```

Is this also available for the collection-provider moderator?

```suggestion
    This information is only available to an admin on the Collection Submission in question.
```